main/src/main/java/org/mobilitydata/gtfsvalidator/validator/StopZoneIdValidator.java

```diff
@@ -71,14 +71,14 @@ private boolean isStationOrEntrance(GtfsStop stop) {
    * A {@code GtfsShape} should be referred to at least once in {@code GtfsTripTableContainer}
    * station).
    *
-   * <p>Severity: {@code SeverityLevel.WARNING} - Will be upgraded to {@code SeverityLevel.ERROR}
+   * <p>Severity: {@code SeverityLevel.ERROR}
```

Comment:
We don't have statistics for this rule at Google yet. How many of your feeds break this rule?

Reply:
As suggested, this PR has been split into different PRs: one for each notice (see #1023). As of today, we do not have the statistics yet for GTFS datasets, but will update when we have them.